applications/nrf5340_audio/src/bluetooth/Kconfig

```diff
@@ -23,6 +23,14 @@ config CODEC_CAP_COUNT_MAX
 	help
 	  Max number of codec capabilities to store per stream.
 
+config EXT_ADV_BUF_MAX
+	int "Max number of advertising data parameters"
+	default 10
+
+config EXT_ADV_UUID_BUF_MAX
+	int "Max number of UUIDs to add to the ext adv"
+	default 10
+
```

Comment:
Would EXT_ADV_BUF_MAX and EXT_ADV_UUID_BUF_MAX always be the same? If yes, we can remove one of it.

Reply:
They will/may not be the same.